### ISSUE TITLE
bug, syntax error not reported, interfaces

### ISSUE THREAD

**Issue by larshp** [abaplint/abaplint#2083]
```abap
INTERFACE lif1.
  METHODS foo.
ENDINTERFACE.

INTERFACE lif2.
  METHODS bar.
ENDINTERFACE.

CLASS bar DEFINITION.
  PUBLIC SECTION.
    METHODS run.
ENDCLASS.

CLASS bar IMPLEMENTATION.
  METHOD run.
    DATA ref TYPE REF TO lif2.
    ref->bar( ).
    ref->lif1~foo( ). " <- this should be a syntax error
  ENDMETHOD.
ENDCLASS.
```
